configure.ac

```diff
@@ -344,7 +344,9 @@ OMPI_CHECK_PORTALS4([portals4],
     [transport_portals4="yes"],
     [transport_portals4="no"])
 
-OPAL_CHECK_OFI([ofi],
+OPAL_CHECK_OFI
+
+AS_IF([test "$opal_ofi_happy" = "yes"],
     [transport_ofi="yes"],
     [transport_ofi="no"])
 
```

Comment:
@kholland-intel Thanks for the changes.. Just checking, did commenting out the PMIx check resolve the error on Travis and locally?

Reply:
Yes, after commenting out the code block which required PMIx the error was resolved both locally and on Travis. The code was commented out on lines 136-140 of opal_check_ofi.m4: